sdk/storage/azure-storage-queues/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 12.0.0-beta.1 (Unreleased)
+
+### New Features
+
+- Nothing yet.
```

Comment:
```suggestion
## 12.0.0-beta.1 (Unreleased)

- Initial release
```

suggestion; This is the phrasing we used for Core's initial release.

Reply:
It's going to be rewritten anyway when we ship the first release. But fine, let me commit your suggestion,